db/db_impl/db_impl_open.cc

```diff
@@ -368,7 +370,7 @@ Status DBImpl::Recover(
     if (!s.ok()) {
       return s;
     }
-
+    
```

Comment:
White spaces in empty line should be removed. You can run `make format`.

Reply:
`make format` reports an error in `clang-format-diff.py`:
`TypeError: initial_value must be str or None, not bytes`.
Do I need to pass in any arguments?